packages/amplify-category-auth/src/provider-utils/awscloudformation/index.js

```diff
@@ -294,23 +295,32 @@ function getRequiredParamsForHeadlessInit(projectType, previousValues) {
 async function console(context, amplifyMeta) {
   const cognitoOutput = getCognitoOutput(amplifyMeta);
   if (cognitoOutput) {
-    const { Region } = amplifyMeta.providers.awscloudformation;
+    const { AmplifyAppId, Region } = amplifyMeta.providers.awscloudformation;
     if (cognitoOutput.UserPoolId && cognitoOutput.IdentityPoolId) {
+      let choices = ['User Pool', 'Identity Pool', 'User Pool and Identity Pool'];
+      let adminOption = doAdminCredentialsExist(AmplifyAppId);
+      if (adminOption) {
+        choices = ['Amplify admin UI', ...choices];
+      }
       const answer = await inquirer.prompt({
         name: 'selection',
         type: 'list',
         message: 'Which console',
-        choices: ['User Pool', 'Identity Pool', 'Both'],
-        default: 'Both',
+        choices,
+        default: adminOption ? 'Amplify admin UI' : 'User Pool and Identity Pool',
```

Comment:
nit can we make some constants / an enum for these strings. Too easy to make a typo or something like this